config/locales/simple_form.fr.yml

```diff
@@ -1,26 +1,42 @@
 ---
 fr:
   simple_form:
+    hints:
+      defaults:
+        avatar: Au format PNG, GIF ou JPG. 2Mo maximum. Sera réduit à 120x120px
+        display_name: 30 caractères maximum
+        header: Au format PNG, GIF ou JPG. 2Mo maximum. Sera réduit à 700x335px
+        locked: Vous devrez approuver chaque abonné⋅e et vos statuts ne s'afficheront qu'à vos abonné⋅es
+        note: 160 caractères maximum
+      imports:
+        data: Un fichier CSV généré par une autre instance de Mastodon
     labels:
       defaults:
         avatar: Image de profil
         confirm_new_password: Confirmation du nouveau mot de passe
         confirm_password: Confirmation du mot de passe
         current_password: Mot de passe actuel
+        data: Données
         display_name: Nom public
         email: Adresse courriel
         header: Image d’en-tête
         locale: Langue
+        locked: Rendre le compte privé
         new_password: Nouveau mot de passe
         note: Présentation
+        otp_attempt: Code d'identification à deux facteurs
         password: Mot de passe
+        setting_default_privacy: Confidentialité des statuts
+        type: Type d'import
         username: Identifiant
       interactions:
         must_be_follower: Masquer les notifications des personnes qui ne vous suivent pas
         must_be_following: Masquer les notifications des personnes que vous ne suivez pas
       notification_emails:
+        digest: Envoyer des emails récapitulatifs
         favourite: Envoyer un courriel lorsque quelqu’un ajoute mes statut à ses favoris
```

Comment:
It's not "mes statut" but "mes statuts"
  favourite: Envoyer un courriel lorsque quelqu’un ajoute mes statuts à ses favoris